devfile.yaml

```diff
@@ -31,7 +31,31 @@ commands:
       label: "[UD] build"
       component: tools
       workingDir: ${PROJECTS_ROOT}/che-dashboard
-      commandLine: "yarn build"
+      commandLine: "yarn --cwd packages/common build && yarn --cwd packages/dashboard-frontend build:dev && yarn --cwd packages/dashboard-backend build:dev"
+      group:
+        kind: build
+  - id: watchfrontend
+    exec:
+      label: "[UD] watch frontend"
+      component: tools
+      workingDir: ${PROJECTS_ROOT}/che-dashboard
```

Comment:
just fyi: `${PROJECT_SOURCE}` should return the same path as `${PROJECTS_ROOT}/che-dashboard`. Shorter and project name agnostic )